models.js

```diff
@@ -72,16 +98,16 @@ debuglog = function () {
             };
             return toFixed(this.value.toString());
         };
+        this.toHashtag = this.toXPath;
```

Comment:
I did this so that it would be possible to be able to translate XPath to hashtags. It'll be helpful for the first pass in Vellum when we want to hashtag existing /data/ references.

It could also be helpful in the future if someone types in something like `instance('casedb')/cases/cases[@case_id=instance('commcaresession')/blah/blah]/property` and we could pop up and say "Did you know you can use `#case/type/property`?"